analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseUnixEpoch.vb

```diff
@@ -8,6 +8,8 @@ Public Class Program
     Private ReadOnly EpochOff As DateTimeOffset = New DateTimeOffset(1970, 1, 1, 0, 0, 0, 0, TimeSpan.Zero) ' Noncompliant {{Use "DateTimeOffset.UnixEpoch" instead of creating DateTimeOffset instances that point to the unix epoch time}}
     '                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+    Private Const EpochTicks As Long = 621355968000000000
```

Comment:
I think you can add test cases with hex numbers and also maybe one with decimal numbers that contain underscores.
Both should work. It would be there just to document what we cover.

Reply:
Same applies for both languages.